Cargo.toml

```diff
@@ -23,6 +23,7 @@ tokio = "1"
 
 # Filesystem
 home = "0.5"
+xdg = "2.5"
```

Comment:
I also see now that the `known_folders` dependency is missing which breaks compilation on Windows. But as this PR is macOS-specific, I'll fix that in a separate PR.